dist/main/index.js

```diff
@@ -75996,22 +75988,20 @@ class UpdaterBuilder {
                     `HTTP_PROXY=${proxyUrl}`,
                     `https_proxy=${proxyUrl}`,
                     `HTTPS_PROXY=${proxyUrl}`,
+                    `UPDATER_ONE_CONTAINER=1`,
```

Comment:
What does this do? And who's the consumer? I don't see it anywhere else in this repo?

Because if `dependabot-action` is the only place using two containers (I don't know if it is), then since it's now one, does that mean this setting should be nuked instead?

Reply:
It triggers [this behavior](https://github.com/dependabot/dependabot-core/blob/8a7113652960b117de8d05d4ec4f8fecc1931b5c/updater/lib/dependabot/file_fetcher_job.rb#L41-L49) which writes the input for `update_files` rather than having something outside of the container do it.